### ISSUE TITLE
Cannot use `select2` within modal

### ISSUE THREAD

**Issue by atd** [Hirurg103/capybara_select2#1]
This code fails:

```
within('#someModal') do
  select2 'some_tag', from 'Tags', tag: true
end
```

is failing with:

```
Unable to find visible css ".select2-container-active .select2-result,.select2-drop-active .select2-result,.select2-results .select2-results
__option" with text "some_tag" within #<Capybara::Node::Element tag="div" path="/html/body/div[@id='modalBlock']/div[@id='someModal']"
```

This is because the selected option is added to a container outside the modal

Maybe https://github.com/Hirurg103/capybara_select2/blob/master/lib/capybara_select2/helpers.rb#L42 should search from the body?

Thank you for reviving this gem!

**Comment by Hirurg103** [comment]
Hi @atd 

I updated the gem but forgot to update the README

In the latest version (0.2.0) you need to specify select 2 version which you are using. See [Configuration](https://github.com/Hirurg103/capybara_select2#configuration)

Try to upgrade the gem and configure version of select2 which you are you using

**Comment by Hirurg103** [comment]
@atd if it still doesn't work could you please specify which modal you are using and attach screenshots  or markdown if possible

**Comment by Hirurg103** [comment]
> Maybe https://github.com/Hirurg103/capybara_select2/blob/master/lib/capybara_select2/helpers.rb#L42 should search from the body?

Yes, you are right. I see the issue now. I will fix it shortly

**Comment by Hirurg103** [comment]
@atd this but should be fixed in version `0.2.1`
